src/utils/CharactersPageUtils.js

```diff
@@ -0,0 +1,86 @@
+import { nanoid } from "@reduxjs/toolkit"
+
+export const prepareCharacterForFormReset = ({
+  name,
+  eyeColor,
+  height,
+  gender,
+  birthYear,
+  homeworld,
+  species,
+  filmConnection,
+}) => {
+  return {
+    name,
+    eyeColor,
+    height: height || "",
+    gender:
+      gender !== "n/a"
+        ? {
+            value: gender,
+            label: gender,
+          }
+        : null,
```

Comment:
Please define this in a variable above and assign it to a property in the object being returned